README.md

```diff
@@ -1,64 +1,316 @@
+1. On the top bar, click on your account, and then on **Switch Directory**.
+1. Once the *Directory + subscription* pane opens, choose the Active Directory tenant where you wish to register your
+application, from the *Favorites* or *All Directories* list.
+1. Click on **All services** in the menu and choose **Azure Active Directory**.
+
+> In the next steps, you might need the tenant name (or directory name) or the tenant ID (or directory ID). These are
+presented in the **Properties** of the Azure Active Directory window respectively as *Name* and *Directory ID*
+
+#### Register the Web Api app (java-webapi)
+
+1. In the  **Azure Active Directory** pane, click on **App registrations** and choose **New registration**.
+1. Enter a friendly name for the application, for example 'java-webapi', select **Accounts in any organizational directory and personal Microsoft Accounts (e.g. Skype, Xbox, Outlook.com)**.
+1. Click **Register** to register the application.
+1. On the left hand menu, click on **Overview** and :
+    - copy **Application (client) ID**
+    - copy **Directory (tenant) ID**
+    - You'll need both of these values later to configure the project, so put them in a safe place.
+
+1. Select **Certificates & Secrets** blade and click on `New client secret` in the **Client Secrets** section:
+
+   - Type a key description (for instance `app secret`).
+   - Select one of the provided key durations as per your security needs.
+   - When you save this page, the key value will be displayed. Copy the value and save for later to configure the project. This key value will not be displayed again, nor retrievable by any other means, so record it as soon as it is visible in the Azure portal.
+
+1. In the Application menu blade select **Expose an API**, click on **Add a scope**
+    - Accept the proposed **Application ID URI** (api://{clientId}) by selecting **save and continue**.
+    - For **scope name** use **access_as_user**
+    - Select **Admins and users** for **who can consent**.
+    - In **Admin consent display name** type `Access Webapi as a user`
+    - In **Admin consent description** type `Allow the application to access Webapi on behalf of the signed-in user.`
+    - In **User consent display name** type `Access Webapi as a user`
+    - In **User consent description** type `Allow the application to access Webapi on your behalf.`
+    - Keep **State** as **Enabled**
+    - Select **Add scope**
+
+#### Configure the **msal-obo-sample** to use your Azure AD tenant
+
+Open `application.properties` in the src/main/resources folder. Fill in with your tenant and app registration information noted in the above registration step.
+
+- Replace *Enter_the_Tenant_Info_Here* with  Directory (tenant) ID.
+- *Enter_the_Application_Id_here* with the Application (client) ID.
+- *Enter_the_Client_Secret_Here* with the *key value* noted.
+
+#### Register the client Web app (Webapp-Openidconnect)
```

Comment:
Replace the title Webapp-Openidconnect, with the name that you re suggesting in the step

Reply:
replaced with the one I used.